source/component/PasDoc_Scanner.pas

```diff
@@ -73,13 +74,16 @@   EInvalidIfCondition = class(EPasDoc);
 
   { This class scans one unit using one or more @link(TTokenizer) objects
     to scan the unit and all nested include files. }
+
+  { TScanner }
```

Comment:
Remove this extra `{ TScanner }` comment. 

It was likely added by Lazarus Code Tools. We advise to turn off in Lazarus IDE the "Header comment for class", see also https://github.com/pasdoc/pasdoc/issues/149#issuecomment-1308837496 . This comment doesn't really add anything useful IMHO, and it obscures the important API documentation above (`{ This class scans....`).

Reply:
Yes, that's probably an auto-comment. I've removed it.

Thanks for the link, I've updated the configuration. It is not useful and also it get in the way of the documentation generated by `pasdoc`.